sdk/devcenter/Azure.Developer.DevCenter/tests/DevBoxClientTests.cs

```diff
@@ -0,0 +1,415 @@
+            {
+                numberOfReturnedDevBoxes++;
+                JsonElement devBoxResponseData = JsonDocument.Parse(devBoxData.ToStream()).RootElement;
+
+                if (!devBoxResponseData.TryGetProperty("name", out var devBoxNameJson))
+                {
+                    Assert.Fail("The JSON response received from the service does not include the necessary property.");
+                }
+
+                string devBoxName = devBoxNameJson.ToString();
+                Assert.AreEqual(devBoxName, DevBoxName);
+            }
+
+            Assert.AreEqual(1, numberOfReturnedDevBoxes);
+        }
+
+        [RecordedTest]
+        public async Task GetPoolSucceeds()
+        {
+            Response getPoolResponse = await _devBoxesClient.GetPoolAsync(TestEnvironment.ProjectName, TestEnvironment.PoolName);
+            JsonElement getPoolData = JsonDocument.Parse(getPoolResponse.ContentStream).RootElement;
+
+            if (!getPoolData.TryGetProperty("name", out var poolNameJson))
+            {
+                Assert.Fail("The JSON response received from the service does not include the necessary property.");
+            }
+
+            string poolName = poolNameJson.ToString();
+            Assert.AreEqual(poolName, TestEnvironment.PoolName);
+        }
+
+        [RecordedTest]
+        public async Task GetPoolsSucceeds()
+        {
+            var numberOfReturnedPools = 0;
+            await foreach (BinaryData poolData in _devBoxesClient.GetPoolsAsync(TestEnvironment.ProjectName))
+            {
+                numberOfReturnedPools++;
+                JsonElement getPoolsResponseData = JsonDocument.Parse(poolData.ToStream()).RootElement;
+
+                if (!getPoolsResponseData.TryGetProperty("name", out var poolNameJson))
+                {
+                    Assert.Fail("The JSON response received from the service does not include the necessary property.");
+                }
+
+                string poolName = poolNameJson.ToString();
+                Assert.AreEqual(poolName, TestEnvironment.PoolName);
+            }
+
+            Assert.AreEqual(1, numberOfReturnedPools);
+        }
+
+        [RecordedTest]
+        public async Task GetSchedulesSucceeds()
+        {
+            var numberOfReturnedSchedules = 0;
+            await foreach (BinaryData scheduleData in _devBoxesClient.GetSchedulesAsync(TestEnvironment.ProjectName, TestEnvironment.PoolName))
+            {
+                numberOfReturnedSchedules++;
+                JsonElement getSchedulesResponseData = JsonDocument.Parse(scheduleData.ToStream()).RootElement;
+
+                if (!getSchedulesResponseData.TryGetProperty("name", out var poolNameJson))
+                {
+                    Assert.Fail("The JSON response received from the service does not include the necessary property.");
+                }
+
+                string poolName = poolNameJson.ToString();
```

Comment:
Rather than `poolName`, shouldn't this be `scheduleName`?

Reply:
Fixed.